sub-packages/bionemo-evo2/src/bionemo/evo2/run/train.py

```diff
@@ -758,12 +1037,26 @@ def train(args: argparse.Namespace) -> nl.Trainer:
         )
     )
     nemo_logger = setup_nemo_lightning_logger(
-        root_dir=args.result_dir,
+        root_dir=args.results_dir,
```

Comment:
`result-dir` is more consistent with other training scripts, so we made that change, can you change it back to result_dir?